quickstart/101-analysis-services-create/variables.tf

```diff
@@ -10,7 +10,7 @@ variable "resource_group_name_prefix" {
   description = "Prefix of the resource group name that's combined with a random ID so name is unique in your Azure subscription."
 }
 
-variable "azurerm_analysis_services_server_name_prefix" {
+variable "analysis_services_server_name_prefix" {
```

Comment:
This variable is an unused variable now, I think we should just delete it.

Reply:
Thanks. Fixed.